operators/pkg/instance-controller/container_logic.go

```diff
@@ -146,6 +147,28 @@ func buildContainerInstanceDeploymentSpec(
 			// LivenessProbe:   &tigerVncProbe,
 			ReadinessProbe: &tigerVncProbe,
 		},
+		{
+			Name:      "filebrowser",
+			Image:     o.FileBrowserImg + ":" + o.FileBrowserImgTag,
+			Resources: buildResRequirements(0.1, 50, resource.MustParse("100Mi")), // actual: ~10MiB
+			Args: []string{
+				"--port=" + fmt.Sprintf("%d", fileBrowserPort),
+				"--root=" + mountPath,
+				"--baseurl=/" + urlUUID + "/mydrive",
+				"--database=/tmp/database.db",
+				"--noauth=true",
+			},
+			SecurityContext: &contSecCtx,
+			Ports: []v1.ContainerPort{{
+				ContainerPort: fileBrowserPort,
```

Comment:
To expand on this, I would set the FileBrowser's TargetPort field of service.Spec.Ports to the same name — at the moment, it is omitted to match the Port field.

Reply:
Well, that would mean to have yet another variable in common to pass between common_logic.go and container_logic.go files, which may prove to be unwieldy.

Reply:
Please let me know if I just should stick with your proposed solution.

Reply:
Not sure if I understood correctly your comment, but currently the port is hard coded, so switching to names would make no difference

Reply:
To explain myself better, I will quote the docs about service TargetPort field: "[...] If this is a string, it will be looked up as a named port in the target Pod's container ports.".

Reply:
Ok, I think I got your message. Generally speaking, named ports in services are slightly better, as more meaningful than just numbers, However, it is not a big deal in this case, you can stick to the actual approach.